src/core/ProductTile/__snapshots__/ProductTile.stories.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Components/Product Tile ProductTileWithOverriddenStylesAndClick smoke-test 1`] = `
-<div class="rounded-lg p-12 flex flex-col gap-8 transition-colors bg-neutral-100 dark:bg-neutral-1200 hover:bg-neutral-200 dark:hover:bg-neutral-1100 bg-pink-200 dark:bg-pink-800 hover:bg-pink-600 dark:hover:bg-pink-400 cursor-pointer">
+<div class="rounded-lg p-12 flex flex-col gap-8 transition-colors bg-pink-200 dark:bg-pink-800 hover:bg-pink-600 dark:hover:bg-pink-400 cursor-pointer">
```

Comment:
This diff shows a good example of `tailwind-merge` in play. On the left, we have a bunch of `bg` classes which are superseded by the classes from a `className` prop - they have been stripped away to leave only the classes we want